### DISCUSSION TITLE
Add the ability to track the visit times of the short url model?

### DISCUSSION THREAD

**Discussion by codesinging** [ash-jc-allen/short-url#87] [Ideas]
Thank you for the awesome package. 
I need to track the visit count of the short url model in my project.
I was wondering if you were considering adding this feature?

**Comment by ash-jc-allen** [comment]
Hey! Thanks, I'm glad that you like the package. If I'm understanding your use case right at the moment, you should already be able to do this.

If you're using the default controller that comes with the package, each visit to a short URL will be stored in the `short_url_visits` table and can be accessed using the `ShortURLVisit` model. So, on the `ShortURL` model, you can call `$shortUrl->visits` to get all of the visits for a particular URL.

Hopefully this helps a little bit? 😄
